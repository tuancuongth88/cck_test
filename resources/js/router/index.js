import Vue from 'vue';
import VueRouter from 'vue-router';

Vue.use(VueRouter);

import homeManagement from './modules/homeManagement.js';
import onGoingJob from './modules/showMoreOnGoingJob.js';
import hrManagement from './modules/hrManagement.js';
import hrOrganization from './modules/hrOrganization.js';
import companyManagement from './modules/companyManagement.js';
import matchingManagement from './modules/matchingManagement.js';
import jobManagement from './modules/jobManagement.js';
import jobSearch from './modules/jobSearch.js';
import hrSearch from './modules/hrSearch.js';
import favorite from './modules/favorite.js';
import changePassword from './modules/changePassword.js';

export const constantRoutes = [
  {
    path: '/login',
    name: 'Login',
    component: () => import(/* webpackChunkName: "Login" */ '@/pages/Login/index.vue'),
    meta: {
      title: 'ROUTER_LOGIN',
    },
    hidden: true,
  },
  // Link follow gmail from BE
  {
    path: '/api/auth/password-reset',
    name: 'Reset password confirm',
    component: () => import('@/pages/PassWords/ResetPassWord/index.vue'),
    meta: {
      title: 'RESET_PASS_WORD_TITLE',
    },
    hidden: true,
  },
  {
    path: '/forget-password',
    name: 'Reset password',
    component: () => import('@/pages/PassWords/ForgetPassWord/index.vue'),
    meta: {
      title: 'RESET_PASS_WORD_TITLE',
    },
    hidden: true,
  },
  {
    path: '/human-resourse-register',
    name: 'human-resourse-register',
    component: () => import('@/pages/RegisterHrOrigin/index.vue'),
    meta: {
      title: 'RegisterHrOrigin',
    },
    hidden: true,
  },
  {
    path: '/company-register',
    name: 'company-register',
    component: () => import('@/pages/RegisterCompany/index.vue'),
    meta: {
      title: 'Company register',
    },
    hidden: true,
  },
  // {
  //   path: '/not-permission',
  //   component: () => import('@/pages/Error/NotPermission.vue'),
  //   meta: {
  //     title: 'Permission not Found',
  //   },
  //   hidden: true,
  // },
];

export const asyncRoutes = [
  homeManagement,
  onGoingJob,
  hrSearch,
  hrManagement,
  hrOrganization,
  companyManagement,
  jobManagement,
  jobSearch,
  matchingManagement,
  favorite,
  changePassword,
  {
    path: '/',
    redirect: { name: 'Home' },
    hidden: true,
  },
];

const createRouter = () => new VueRouter({
  mode: 'history',
  scrollBehavior: () => ({ y: 0 }),
  routes: constantRoutes,
});

const router = createRouter();

export function resetRouter() {
  const newRouter = createRouter();
  router.matcher = newRouter.matcher;
}

export default router;
