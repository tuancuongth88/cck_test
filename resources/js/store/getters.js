const getters = {
  role: state => state.user.role,
  etc: state => state.etc.etcOrigin,
  loading: state => state.app.loading,
  work_id: state => state.job.work_id,
  profile: state => state.user.profile,
  language: state => state.app.language,
  tabIndex: state => state.app.tabIndex,
  name: state => state.user.profile.name,
  token: state => state.user.access_token,
  email: state => state.user.profile.email,
  routes: state => state.permission.routes,
  permission: state => state.user.permission,
  isReload: state => state.permission.isReload,
  message_id: state => state.message.message_id,
  expToken: state => state.user.profile.expToken,
  addRoutes: state => state.permission.addRoutes,
  listUser: state => state.userManagement.listUser,
  permissionCheck: state => state.user.profile.type,
  is_filled_data: state => state.job.is_filled_data,
  permissionRoutes: state => state.permission.routes,
  degitacho: state => state.degitacho.degitachoOrigin,
  list_selected_hr: state => state.job.list_selected_hr,
  list_disabled_hr: state => state.job.list_disabled_hr,
  currentRoutes: state => state.permission.currentRoutes,
  job_search_data: state => state.jobSearch.job_search_data,
  searchParams: state => state.hrSearchQuery.searchParams,
  list_hr_information: state => state.job.list_hr_information,
  setOpen: state => state.app.setOpen,
  routerInfo: state => state.routerParam.routerInfo,
  revertRouter: state => state.routerParam.revertRouter,
};

export default getters;
