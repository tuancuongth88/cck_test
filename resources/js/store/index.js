import Vue from 'vue';
import Vuex from 'vuex';

import app from './modules/app';
import job from './modules/job';
import user from './modules/user';
import message from './modules/message';
import jobSearch from './modules/jobSearch';
import permission from './modules/permission';
import hrSearchQuery from './modules/hrSearchQuery';
import userManagement from './modules/userManagement';
import routerParam from './modules/routerParam';

import getters from './getters';

Vue.use(Vuex);

const modules = {
  app,
  job,
  user,
  message,
  jobSearch,
  permission,
  hrSearchQuery,
  userManagement,
  routerParam,
};

const store = new Vuex.Store({
  modules,
  getters,
});

export default store;
