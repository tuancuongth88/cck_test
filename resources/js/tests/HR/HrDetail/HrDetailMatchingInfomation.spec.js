import store from '@/store';
import router from '@/router';
import tab5MatchingSituation from '@/pages/Hr/Detail/TabEMatchingSituation';
import { mount, createLocalVue, shallowMount } from '@vue/test-utils';

describe('TEST SCREEN HR Detail Basic Infomation', () => {
  const DATA = {
    id: 11,
    country_id: 1,
    hr_organization_id: 1,
    user_id: '5',
    full_name: 'Hoa 11',
    full_name_ja: 'Hoa 11ja',
    gender: 1,
    date_of_birth: '2000-05-11',
    work_form: 1,
    preferred_working_hours: 8,
    japanese_level: 1,
    final_education_date: '2023-08-01',
    final_education_classification: 1,
    final_education_degree: 1,
    major_classification_id: 1,
    middle_classification_id: 2,
    personal_pr_special_notes: 'test personal pr special note',
    remarks: 'remarks',
    telephone_number: '+84 0987654321',
    mobile_phone_number: '+81 0123456789',
    mail_address: 'vuhoa@gmail.com',
    address_city: 'Ha Noi city',
    address_district: '266 Doi Can address district',
    address_number: '266 address number',
    address_other: 'other address',
    hometown_city: 'Ha Tay hometown',
    home_town_district: 'Ung Hoa homtown district',
    home_town_number: '144 homtown number',
    home_town_other: 'other homtown',
    file_cv_id: 49,
    file_job_id: 50,
    file_others: '[{"name":null,"file_id":null,"file_path":null}]',
    status: 1,
    created_by: 1,
    updated_by: null,
    deleted_at: null,
    created_at: 1689214928,
    updated_at: 1689577354,
    entry: [
      {
        id: 1,
        status: 4,
        job_title: 'php',
        company_name: 'City Computer Co., Ltd.',
        updated_at: '2023年07月11日',
      },
      {
        id: 5,
        status: 1,
        job_title: 'php',
        company_name: 'City Computer Co., Ltd.',
        updated_at: '2023年07月11日',
      },
    ],
    offer: [
      {
        id: 2,
        status: 2,
        job_title: 'php',
        company_name: 'City Computer Co., Ltd.',
        updated_at: '2023年07月10日',
      },
      {
        id: 3,
        status: 1,
        job_title: 'Tuyên php',
        company_name: 'City Computer Co., Ltd.',
        updated_at: '2023年07月10日',
      },
    ],
    interview: [
      {
        id: 1,
        status: 2,
        job_title: 'php',
        company_name: 'City Computer Co., Ltd.',
        updated_at: '2023年05月20日',
      },
      {
        id: 6,
        status: 2,
        job_title: 'Tuyên php',
        company_name: 'City Computer Co., Ltd.',
        updated_at: '2023年05月20日',
      },
      {
        id: 11,
        status: 1,
        job_title: 'php',
        company_name: 'City Computer Co., Ltd.',
        updated_at: '2023年07月11日',
      },
    ],
    result: [
      {
        id: 1,
        status: 3,
        job_title: 'php',
        company_name: 'City Computer Co., Ltd.',
        updated_at: '2023年07月20日',
      },
    ],
    fileCV: {
      id: 49,
      file_model: 'test_file.pptx (0.03MB)',
      file_name: 'test_file.pptx',
      file_extension: 'pptx',
      file_path: 'storage/test_file.pptx (0.03MB)/20230717/308ae11a7e6c4b09440972ebfc2e45b5test_file.pptx',
      file_size: '32825',
      item_type: '',
      item_id: null,
      created_at: '2023-07-17T07:02:30.000000Z',
      updated_at: '2023-07-17T07:02:30.000000Z',
      deleted_at: null,
    },
    fileJob: {
      id: 50,
      file_model: 'test_file.pptx (0.03MB)',
      file_name: 'test_file.pptx',
      file_extension: 'pptx',
      file_path: 'storage/test_file.pptx (0.03MB)/20230717/d4a50aa9c61e0e45bd5eecd6a07c3bcbtest_file.pptx',
      file_size: '32825',
      item_type: '',
      item_id: null,
      created_at: '2023-07-17T07:02:33.000000Z',
      updated_at: '2023-07-17T07:02:33.000000Z',
      deleted_at: null,
    },
    is_favorite: 1,
    file_c_v: {
      id: 49,
      file_model: 'test_file.pptx (0.03MB)',
      file_name: 'test_file.pptx',
      file_extension: 'pptx',
      file_path: 'storage/test_file.pptx (0.03MB)/20230717/308ae11a7e6c4b09440972ebfc2e45b5test_file.pptx',
      file_size: '32825',
      item_type: '',
      item_id: null,
      created_at: '2023-07-17T07:02:30.000000Z',
      updated_at: '2023-07-17T07:02:30.000000Z',
      deleted_at: null,
    },
    file_job: {
      id: 50,
      file_model: 'test_file.pptx (0.03MB)',
      file_name: 'test_file.pptx',
      file_extension: 'pptx',
      file_path: 'storage/test_file.pptx (0.03MB)/20230717/d4a50aa9c61e0e45bd5eecd6a07c3bcbtest_file.pptx',
      file_size: '32825',
      item_type: '',
      item_id: null,
      created_at: '2023-07-17T07:02:33.000000Z',
      updated_at: '2023-07-17T07:02:33.000000Z',
      deleted_at: null,
    },
    main_jobs: [
      {
        id: 32,
        hrs_id: 11,
        main_job_career_date_from: '2021-01-01',
        main_job_career_date_to: '2021-02-01',
        department_id: 18,
        job_id: 79,
        detail: 'gsfdfg',
        deleted_at: null,
        created_at: '2023-07-17T07:02:34.000000Z',
        updated_at: '2023-07-17T07:02:34.000000Z',
      },
    ],
  };

  test('Test 1: Test render tab Basic Infomation (component)', async() => {
    const localVue = createLocalVue();
    const wrapper = shallowMount(tab5MatchingSituation, {
      localVue,
      router,
      store,
      stubs: {
        BIcon: true,
      },
    });

    const render_component = wrapper.find('.hr-detail-matching-situation-tab');
    expect(render_component.exists()).toBe(true);

    wrapper.destroy();
  });

  test('Test 2: test render data Basic Infomation', async() => {
    const DATA = {
      data: {
        dataEntries: [],
        dataOffers: [],
        dataInterviews: [],
        dataResults: [],
      },
      detailData: {
        type: Object,
        require: true,
        default: function() {
          return {};
        },
      },
    };

    const localVue = createLocalVue();
    const wrapper = shallowMount(tab5MatchingSituation, {
      localVue,
      router,
      store,
      stubs: {
        BIcon: true,
      },
      mocks: { $t: () => {} },
      propsData: {
        detailData: [],
      },
    });

    expect(wrapper.vm.dataEntries).toEqual(DATA.data.dataEntries);
    expect(wrapper.vm.dataOffers).toEqual(DATA.data.dataOffers);
    expect(wrapper.vm.dataInterviews).toEqual(DATA.data.dataInterviews);
    expect(wrapper.vm.dataResults).toEqual(DATA.data.dataResults);

    expect(wrapper.props().detailData).toEqual([]);

    wrapper.destroy();
  });

  test('Test 3: test render content data Entry Application History', async() => {
    const localVue = createLocalVue();
    const wrapper = mount(tab5MatchingSituation, {
      localVue,
      router,
      store,
      stubs: {
        BIcon: true,
      },
      propsData: {
        detailData: [],
      },
    });

    expect(wrapper.props().detailData).toEqual([]);
    await wrapper.setProps({ detailData: DATA });

    expect(wrapper.props().detailData).toEqual(DATA);

    expect(wrapper).toMatchSnapshot();

    const label_entry_applycation = wrapper.find('.label-entry-applycation');
    expect(label_entry_applycation.exists()).toBe(true);
    expect(label_entry_applycation.text()).toBe('HR_LIST.ENTRY_APPLICATION');

    const status_entry = wrapper.findAll('.status-entry');
    expect(status_entry.exists()).toBe(true);
    expect(status_entry.length).toEqual(2);
    expect(status_entry.at(1).text()).toEqual('申請中');

    const entry_title_company = wrapper.findAll('.entry-title-company');
    expect(entry_title_company.exists()).toBe(true);
    expect(entry_title_company.length).toEqual(2);
    expect(entry_title_company.at(1).text()).toEqual('php/City Computer Co., Ltd.');

    const label_entry_date = wrapper.findAll('.label-entry-date');
    expect(label_entry_date.exists()).toBe(true);
    expect(label_entry_date.length).toEqual(2);
    expect(label_entry_date.at(1).text()).toBe('HR_LIST.ENTRY_DATE：');

    const entry_date = wrapper.findAll('.entry-date');
    expect(entry_date.exists()).toBe(true);
    expect(entry_date.length).toEqual(2);
    expect(entry_date.at(1).text()).toEqual('');

    wrapper.destroy();
  });

  test('Test 4: test render content data Offer History', async() => {
    const localVue = createLocalVue();
    const wrapper = mount(tab5MatchingSituation, {
      localVue,
      router,
      store,
      stubs: {
        BIcon: true,
      },
      propsData: {
        detailData: [],
      },
    });

    expect(wrapper.props().detailData).toEqual([]);
    await wrapper.setProps({ detailData: DATA });

    expect(wrapper.props().detailData).toEqual(DATA);

    // expect(wrapper).toMatchSnapshot();

    const label_entry_applycation = wrapper.find('.label-offer-received');
    expect(label_entry_applycation.exists()).toBe(true);
    expect(label_entry_applycation.text()).toBe('HR_LIST.OFFER_RECEIVED');

    const status_entry = wrapper.findAll('.offer-status');
    expect(status_entry.exists()).toBe(true);
    expect(status_entry.length).toEqual(2);
    expect(status_entry.at(1).text()).toEqual('申請中');

    const entry_title_company = wrapper.findAll('.offer-title-company');
    expect(entry_title_company.exists()).toBe(true);
    expect(entry_title_company.length).toEqual(2);
    expect(entry_title_company.at(1).text()).toEqual('Tuyên php/City Computer Co., Ltd.');

    const label_entry_date = wrapper.findAll('.label-offer-date');
    expect(label_entry_date.exists()).toBe(true);
    expect(label_entry_date.length).toEqual(2);
    expect(label_entry_date.at(1).text()).toBe('HR_LIST.ENTRY_DATE：');

    const entry_date = wrapper.findAll('.offer-date');
    expect(entry_date.exists()).toBe(true);
    expect(entry_date.length).toEqual(2);
    expect(entry_date.at(1).text()).toEqual('');

    wrapper.destroy();
  });

  test('Test 5: test render content data Interview History', async() => {
    const localVue = createLocalVue();
    const wrapper = mount(tab5MatchingSituation, {
      localVue,
      router,
      store,
      stubs: {
        BIcon: true,
      },
      propsData: {
        detailData: [],
      },
    });

    expect(wrapper.props().detailData).toEqual([]);
    await wrapper.setProps({ detailData: DATA });

    expect(wrapper.props().detailData).toEqual(DATA);

    // expect(wrapper).toMatchSnapshot();

    const label_entry_applycation = wrapper.find('.label-arr-interview');
    expect(label_entry_applycation.exists()).toBe(true);
    expect(label_entry_applycation.text()).toBe('HR_LIST.ARRINTERVIEW');

    const status_entry = wrapper.findAll('.interview-status');
    expect(status_entry.exists()).toBe(true);
    expect(status_entry.length).toEqual(3);
    expect(status_entry.at(1).text()).toEqual('オファー承認');

    const entry_title_company = wrapper.findAll('.interview-title-company');
    expect(entry_title_company.exists()).toBe(true);
    expect(entry_title_company.length).toEqual(3);
    expect(entry_title_company.at(1).text()).toEqual('Tuyên php/City Computer Co., Ltd.');

    const label_entry_date = wrapper.findAll('.label-interview-date');
    expect(label_entry_date.exists()).toBe(true);
    expect(label_entry_date.length).toEqual(3);
    expect(label_entry_date.at(1).text()).toBe('HR_LIST.ENTRY_DATE：');

    const entry_date = wrapper.findAll('.interview-date');
    expect(entry_date.exists()).toBe(true);
    expect(entry_date.length).toEqual(3);
    expect(entry_date.at(1).text()).toEqual('');

    wrapper.destroy();
  });

  test('Test 6: test render content data Result History', async() => {
    const localVue = createLocalVue();
    const wrapper = mount(tab5MatchingSituation, {
      localVue,
      router,
      store,
      stubs: {
        BIcon: true,
      },
      propsData: {
        detailData: [],
      },
    });

    expect(wrapper.props().detailData).toEqual([]);
    await wrapper.setProps({ detailData: DATA });

    expect(wrapper.props().detailData).toEqual(DATA);

    // expect(wrapper).toMatchSnapshot();

    const label_entry_applycation = wrapper.find('.label-result');
    expect(label_entry_applycation.exists()).toBe(true);
    expect(label_entry_applycation.text()).toBe('HR_LIST.RESULT');

    const status_entry = wrapper.findAll('.result-status');
    expect(status_entry.exists()).toBe(true);
    expect(status_entry.length).toEqual(1);
    expect(status_entry.at(0).text()).toEqual('内定承諾');

    const entry_title_company = wrapper.findAll('.result-title-company');
    expect(entry_title_company.exists()).toBe(true);
    expect(entry_title_company.length).toEqual(1);
    expect(entry_title_company.at(0).text()).toEqual('php/City Computer Co., Ltd.');

    const label_entry_date = wrapper.findAll('.label-result-date');
    expect(label_entry_date.exists()).toBe(true);
    expect(label_entry_date.length).toEqual(1);
    expect(label_entry_date.at(0).text()).toBe('HR_LIST.ENTRY_DATE：');

    const entry_date = wrapper.findAll('.result-date');
    expect(entry_date.exists()).toBe(true);
    expect(entry_date.length).toEqual(1);
    expect(entry_date.at(0).text()).toEqual('');

    wrapper.destroy();
  });
});
