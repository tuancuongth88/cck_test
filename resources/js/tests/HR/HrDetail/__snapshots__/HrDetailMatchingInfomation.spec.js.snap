// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEST SCREEN HR Detail Basic Infomation Test 3: test render content data Entry Application History 1`] = `
<div class="hr-detail-matching-situation-tab">
  <div class="hr-detail-list-status-process">
    <div class="hr-detail-list-status-process-item border-t border-l border-r">
      <div class="hr-detail-list-status-process-item--head border-b"><span class="label-entry-applycation">HR_LIST.ENTRY_APPLICATION</span></div>
      <div class="hr-detail-list-status-process--content">
        <div class="hr-detail-list-status">
          <div class="hr-detail-list-status-item">
            <div>
              <div class="hr-detail-list-status-item__status">
                <div class="hr-status-block grey-frame"><span class="one-line-paragraph status-entry">他社内定</span></div>
              </div>
              <div class="hr-detail-list-status-item__candidate-info"><span class="one-line-paragraph entry-title-company">php/City Computer Co., Ltd.</span></div>
            </div>
            <div>
              <div class="hr-detail-list-status-item__entry-date">
                <div class="label-entry-date">HR_LIST.ENTRY_DATE：</div> <span class="one-line-paragraph entry-date"></span>
              </div>
            </div>
          </div>
          <div class="hr-detail-list-status-item">
            <div>
              <div class="hr-detail-list-status-item__status">
                <div class="hr-status-block grey-frame"><span class="one-line-paragraph status-entry">申請中</span></div>
              </div>
              <div class="hr-detail-list-status-item__candidate-info"><span class="one-line-paragraph entry-title-company">php/City Computer Co., Ltd.</span></div>
            </div>
            <div>
              <div class="hr-detail-list-status-item__entry-date">
                <div class="label-entry-date">HR_LIST.ENTRY_DATE：</div> <span class="one-line-paragraph entry-date"></span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="hr-detail-list-status-process-item border-t border-l border-r">
      <div class="hr-detail-list-status-process-item--head border-b"><span class="label-offer-received">HR_LIST.OFFER_RECEIVED</span></div>
      <div class="hr-detail-list-status-process--content">
        <div class="hr-detail-list-status">
          <div class="hr-detail-list-status-item">
            <div>
              <div class="hr-detail-list-status-item__status">
                <div class="hr-status-block red-frame"><span class="one-line-paragraph offer-status">辞退</span></div>
              </div>
              <div class="hr-detail-list-status-item__candidate-info"><span class="one-line-paragraph offer-title-company">php/City Computer Co., Ltd.</span></div>
            </div>
            <div>
              <div class="hr-detail-list-status-item__entry-date">
                <div class="label-offer-date">HR_LIST.ENTRY_DATE：</div> <span class="one-line-paragraph offer-date"></span>
              </div>
            </div>
          </div>
          <div class="hr-detail-list-status-item">
            <div>
              <div class="hr-detail-list-status-item__status">
                <div class="hr-status-block grey-frame"><span class="one-line-paragraph offer-status">申請中</span></div>
              </div>
              <div class="hr-detail-list-status-item__candidate-info"><span class="one-line-paragraph offer-title-company">Tuyên php/City Computer Co., Ltd.</span></div>
            </div>
            <div>
              <div class="hr-detail-list-status-item__entry-date">
                <div class="label-offer-date">HR_LIST.ENTRY_DATE：</div> <span class="one-line-paragraph offer-date"></span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="hr-detail-list-status-process-item border-t border-l border-r">
      <div class="hr-detail-list-status-process-item--head border-b"><span class="label-arr-interview">HR_LIST.ARRINTERVIEW</span></div>
      <div class="hr-detail-list-status-process--content">
        <div class="hr-detail-list-status">
          <div class="hr-detail-list-status-item">
            <div>
              <div class="hr-detail-list-status-item__status">
                <div class="hr-status-block blue-frame"><span class="one-line-paragraph interview-status">オファー承認</span></div>
              </div>
              <div class="hr-detail-list-status-item__candidate-info"><span class="one-line-paragraph interview-title-company">php/City Computer Co., Ltd.</span></div>
            </div>
            <div>
              <div class="hr-detail-list-status-item__entry-date">
                <div class="label-interview-date">HR_LIST.ENTRY_DATE：</div> <span class="one-line-paragraph interview-date"></span>
              </div>
            </div>
          </div>
          <div class="hr-detail-list-status-item">
            <div>
              <div class="hr-detail-list-status-item__status">
                <div class="hr-status-block blue-frame"><span class="one-line-paragraph interview-status">オファー承認</span></div>
              </div>
              <div class="hr-detail-list-status-item__candidate-info"><span class="one-line-paragraph interview-title-company">Tuyên php/City Computer Co., Ltd.</span></div>
            </div>
            <div>
              <div class="hr-detail-list-status-item__entry-date">
                <div class="label-interview-date">HR_LIST.ENTRY_DATE：</div> <span class="one-line-paragraph interview-date"></span>
              </div>
            </div>
          </div>
          <div class="hr-detail-list-status-item">
            <div>
              <div class="hr-detail-list-status-item__status">
                <div class="hr-status-block blue-frame"><span class="one-line-paragraph interview-status">書類通過</span></div>
              </div>
              <div class="hr-detail-list-status-item__candidate-info"><span class="one-line-paragraph interview-title-company">php/City Computer Co., Ltd.</span></div>
            </div>
            <div>
              <div class="hr-detail-list-status-item__entry-date">
                <div class="label-interview-date">HR_LIST.ENTRY_DATE：</div> <span class="one-line-paragraph interview-date"></span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="hr-detail-list-status-process-item border-t border-l border-r border-b">
      <div class="hr-detail-list-status-process-item--head border-b"><span class="label-result">HR_LIST.RESULT</span></div>
      <div class="hr-detail-list-status-process--content">
        <div class="hr-detail-list-status">
          <div class="hr-detail-list-status-item">
            <div>
              <div class="hr-detail-list-status-item__status">
                <div class="hr-status-block blue-frame"><span class="one-line-paragraph result-status">内定承諾</span></div>
              </div>
              <div class="hr-detail-list-status-item__candidate-info"><span class="one-line-paragraph result-title-company">php/City Computer Co., Ltd.</span></div>
            </div>
            <div>
              <div class="hr-detail-list-status-item__entry-date">
                <div class="label-result-date">HR_LIST.ENTRY_DATE：</div> <span class="one-line-paragraph result-date"></span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
